packages/terra-compact-interactive-list/src/subcomponents/Cell.jsx

```diff
@@ -1,15 +1,37 @@
-import React, { useContext } from 'react';
+import React, { useContext, useState, useEffect } from 'react';
+import * as KeyCode from 'keycode-js';
 import PropTypes from 'prop-types';
 import classNames from 'classnames/bind';
 import ThemeContext from 'terra-theme-context';
 import { checkIfColumnIsResponsive } from '../utils/utils';
+import { getFocusableElements } from '../utils/keyHandlerUtils';
 import formattedColumnShape from '../proptypes/formattedColumnShape';
 import { widthUnitTypes, alignTypes } from '../utils/constants';
 import styles from './Cell.module.scss';
 
 const cx = classNames.bind(styles);
 
 const propTypes = {
+  /**
+   * String identifier of the row in which the Cell will be rendered.
+   */
+  rowId: PropTypes.string.isRequired,
+
+  /**
+   * The cell's row position in the table. This is zero based.
+   */
+  rowIndex: PropTypes.number,
+
+  /**
+   * The cell's column position in the table. This is zero based.
+   */
+  columnIndex: PropTypes.number,
```

Comment:
Shouldn't the cell also have a column id?

Reply:
It does, 2 lines above, line 18 is  `rowId: PropTypes.string.isRequired`